src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background: url(../images/star.svg) center no-repeat;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} .stars__star:nth-child(-n + #{$i}) {
+      background: url(../images/star-active.svg) center no-repeat;
```

Comment:
You can change only background-image on hover in order not to duplicate other properties